src/com/yetanalytics/lrs_admin_ui/handlers.cljs

```diff
@@ -113,11 +121,13 @@
  :session/authenticate
  global-interceptors
  (fn [{{server-host ::db/server-host
+        proxy-path  ::db/proxy-path
         :as db} :db} _]
    {:http-xhrio {:method          :post
                  :uri             (httpfn/serv-uri
                                    server-host
-                                   "/admin/account/login")
+                                   "/admin/account/login"
+                                   :proxy-path proxy-path)
```

Comment:
Are there any calls to `serv-uri` or `build-xapi-uri` that _don't_ pass in `proxy-path`? If so you might get away with making `proxy-path` a regular arg instead of a kwarg.

Reply:
4b1b91c32b43cc10aa2d3d44fba800b69e59914b

Reply:
disregard print in above commit, already rectified